ingestion/functions/parsing/taiwan/taiwan.py

```diff
@@ -0,0 +1,122 @@
+import os
+import sys
+from datetime import datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+    import parsing_lib
+
+TAIWAN_LOCATION = {
+    "country": "Taiwan",
+    # Go to wikipedia to debate please, thank you.
+    "geoResolution": "Country",
+    "name": "Taiwan",
+    "geometry": {
+        "longitude": "120.930229378541",
+        "latitude": "23.7779779950014"
+    }
+}
+
+
+def convert_date(raw_date):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+
+    Date filtering API expects mm/dd/YYYYZ format.
+    """
+    date = datetime.strptime(raw_date, "%Y/%m/%d")
+    return date.strftime("%m/%d/%YZ")
+
+
+def convert_gender(raw_gender: str):
+    if raw_gender == "男":
+        return "Male"
+    elif raw_gender == "女":
+        return "Female"
+    else:
+        raise ValueError(f'Unhandled gender: {raw_gender}')
+
+
+def convert_demographics(gender: str, age: str):
+    demo = {}
+    if gender:
+        demo["gender"] = convert_gender(gender)
+    if age:
+        # Brackets are like "35-39" or "over 85".
+        if '-' in age:
+            start, _, end = age.partition("-")
+            demo["ageRange"] = {
+                "start": float(start),
+                "end": float(end),
+            }
+        elif age.isdigit():
+            demo["ageRange"] = {
+                "start": float(age),
+                "end": float(age),
+            }
+        elif age == "70+":
+            demo["ageRange"] = {
+                "start": 70.0,
```

Comment:
Stephen requested to set the end age to the max age (120) for open ended ranges: https://github.com/globaldothealth/list/issues/459

Reply:
waaat, I changed that yesterday as per ab's remark
https://github.com/globaldothealth/list/pull/974/files

I think it's cleaner, what's the rationale? I don't understand/agree with his remark: "If we leave it open ended it could lead to bad or
incomplete data and age is a key demographic data point".

Reply:
Stephen is out until Monday, I put the 120 as max here and for Estonia as well.